packages/vitest/src/node/project.ts

```diff
@@ -390,6 +390,12 @@ export class TestProject {
     this.testFilesList?.push(testPath)
   }
 
+  removeCachedTestFile(testPath: string): void {
```

Comment:
this is an internal method

it should start with _ and have `@internal` jsdoc

Reply:
Thanks! I've updated it.